src/test/java/uk/gov/hmcts/sptribs/testutil/TestConstants.java

```diff
@@ -25,6 +25,9 @@ public final class TestConstants {
     public static final String HEARING_OPTIONS_REGION_DATA_MID_EVENT_URL = "/callbacks/mid-event?page=hearingOptionsRegionData";
     public static final String EDIT_DRAFT_ORDER_MID_EVENT_URL = "callbacks/mid-event?page=editDraftOrderAddDocumentFooter";
     public static final String EDIT_RECORD_LISTING_MID_EVENT_URL = "callbacks/mid-event?page=regionInfo";
+    public static final String ISSUE_DECISION_MID_EVENT_URL = "callbacks/mid-event?page=issueDecisionAddDocumentFooter";
+    public static final String ISSUE_FINAL_DECISION_MID_EVENT_URL = "callbacks/mid-event?page=issueFinalDecisionAddDocumentFooter";
+    public static final String ISSUE_FINAL_DECISION_UPLOAD_MID_EVENT_URL = "callbacks/mid-event?page=issueFinalDecisionUpload";
```

Comment:
is it possible to have these in the functional test folder since that is where they are used?